shopfloor_dangerous_goods_mobile/static/wms/src/css/dangerous_goods.css

```diff
@@ -1,10 +1,12 @@
+.detail-picking-select .has-lq-products .record-name::after,
```

Comment:
hmm I fixed this... AFAIR `record-name` should not  be there anymore.
Can you check pls? And ping me if you don't have findings ;)

Reply:
As discussed, we still have `record-name` in the "picking-select-package-content". I added a TODO to remind us to update it in the future.